docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -124,4 +123,117 @@ An {oldetection} configuration object has the following properties:
 `feature_influence_threshold`:: 
   (double) The minimum {olscore} that a document needs to have in order to 
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+
+[discrete]
+[[regression-resources]]
+==== {regression-cap} configuration objects
+
+A `{regression}` configuration object has the following properties:
+
+`dependent_variable`::
+  (Required, string) Defines which field of the {dataframe} is to be predicted. 
+  This parameter is supplied by field name and must match one of the fields in 
+  the index being used to train. If this field is missing from a document that 
+  row will not be used for training, but a prediction with the trained model 
+  will be generated for it. The data type of the field must be numeric. It is 
+  also known as continuous target variable.  
+
+`eta`::
+  (Optional, double) The shrinkage applied to the weights. Smaller values result 
+  in larger forests which have better generalization error. However, the smaller 
+  the value the longer the training will take and the inference will also be 
+  larger and slower. For more information, see 
```

Comment:
this doesn't read quite right, I'd say instead
"the inference will also be larger and slower" -> "the model will be larger and inference slower"